simp-rake-helpers.gemspec

```diff
@@ -1,4 +1,4 @@
-require 'rake/tasklib'
+#require 'rake/tasklib'
```

Comment:
@heliocentric Did you mean to remove this line, or leave it in?

Reply:
It breaks the gem build unless you run it in bundler, and frankly I couldn't figure out why it was in the gemspec other then being left over after a refactor.

Reply:
I can just completely remove the line, unless there is magic it's doing I didn't see

Reply:
@heliocentric If it still works, nuke it. I just wasn't sure if it was an artifact of debugging or something else.